feat(integrations): Direct users to the reference implementation (#5053)
getsentry/sentry-docs: src/docs/product/integrations/integration-platform/index.mdx

Sentry’s integration platform provides a way for external services to interact with Sentry using [webhooks](/product/integrations/integration-platform/webhooks/), [UI components](/product/integrations/integration-platform/ui-components/), and the [REST API](/api). Integrations using this platform are first-class actors within Sentry.

> ## Quick Start

Reviewer: This is a misnomer, because "Quick Start" generally means "quick start guide" and those are usually meant to get you up and running in a real world situation, not with a demonstration of one. I'm not married to the title I've suggested, but "quick start" is misleading here. 

Less important, I'm also wondering about whether we should be using the word "demo" vs "reference" when referring to this implementation. Seems like people will immediately know what we mean by "demo" but I'm less sure of "reference."? Probably either is fine, but just wanted to raise that.

```suggestion
## Reference Implementation
```

> If you're new to the integration platform, it may help to get started with an example project to demo the different features and data avaiable your application. To help out, we've built a [reference implementation](https://github.com/getsentry/integration-platform-example) which you can use as a starting point.

Reviewer 2: This first sentence is really long. I'll split it into three like: 
```
If you're new to the integration platform, it may help to get started with an example project. To help out, we've built a reference implementation which you can use as a starting point. It demos the different features and data available your application. 
```
also `avaiable` -> `available`.

> If you're new to the integration platform, it may help to get started with an example project. To help out, we've built a [reference implementation](https://github.com/getsentry/integration-platform-example) which you can use as a starting point. It demos the different features and data available your application.

Reviewer: ```suggestion
If you're new to the integration platform, it might help to get started with an example project. To help you out, we've built a [reference implementation](https://github.com/getsentry/integration-platform-example), which you can use as a starting point. It demonstrates the different features and data available for your application:
```

> If you're new to the integration platform, it might help to get started with an example project. To help you out, we've built a [example app](https://github.com/getsentry/integration-platform-example), which you can use as a starting point. It demonstrates the different features and data available for your application:

Reviewer: ```suggestion
If you're new to the integration platform, it might help to get started with an example project. To help you out, we've built an [example application](https://github.com/getsentry/integration-platform-example), which you can use as a starting point. It demonstrates the different features and data available for your application:
```

![Demo kanban application user interface that was built to show integration platform features.](reference-implementation-frontend.png)

> It's a kanban application that uses many of the features available to Sentry integrations and provides starter code in both Python and TypeScript. It also includes a step-by-step guide to setting up your first integration, documentation, and testing advice. Your app will be safely consuming live Sentry data in minutes.

Reviewer: Really want to move the promising out of this phrasing in the last sentence

```suggestion
It's a kanban application that uses many of the features available to Sentry integrations and it provides starter code in both Python and TypeScript. It also includes a step-by-step guide to setting up your first integration, as well as documentation and testing advice. Typically, the app should be safely consuming live Sentry data within minutes.
```

> It's a simple kanban application that leverages many of the features available to Sentry integrations and provides starter code in both Python and TypeScript. It also includes a step-by-step guide to setting up your first integration, a wealth of docs and testing advice, and will have your app safely consuming live Sentry data in minutes!

Reviewer 2: This last sentence looks like an oxfordized list. I'd do:
```
It also includes a step-by-step guide to setting up your first integration, a wealth of docs, and testing advice. Your app will be safely consuming live Sentry data in minutes!
```
